modules/apps/message-boards/message-boards-service/service.xml

```diff
@@ -551,6 +551,7 @@
 	</entity>
 	<exceptions>
 		<exception>BannedUser</exception>
+		<exception>BoardAttachment</exception>
```

Comment:
Question: Should it be called NoSuchBoardAttachment instead?

Reply:
The previous WikiPageAttachment issue, I've created a `NoSuchPageAttachment` and Brian told me to change it `PageAttachment`. That's why I named this exception